app/search.php

```diff
@@ -0,0 +1 @@
+<?php require 'functions.php'; ?>
```

Comment:
It's not in header.php - I moved it out of there because I realised everything was going to send to a POST request to these files, which is where I imagined the functions being called from based on the parameters. Maybe 'search' is different tho, as that actually returns a template